libr/bin/format/mz/mz.c

```diff
@@ -78,6 +75,10 @@ struct r_bin_mz_segment_t * r_bin_mz_get_segments(
 		btree_add (&tree, (void *)&stack_segment, cmp_segs);
 	}
 
+	if (num_relocs * sizeof(*segments) ==  0) {
```

Comment:
Then please do, because I'm quite sure that static analysis tools will yell at you that you're using `sizeof` on a pointer, and it's arch-dependent.